src/main/kotlin/com/terraformation/backend/accelerator/db/ApplicationStore.kt

```diff
@@ -366,6 +367,23 @@ class ApplicationStore(
     }
   }
 
+  private fun updatePrescreenFeedback(applicationId: ApplicationId, feedback: List<String>) {
+    if (feedback.isNotEmpty()) {
+      val feedbackHtml =
```

Comment:
Does this need to be HTML? It may work as you need with regular line break, this is how it is stored for the other feedback columns that use a text area on the frontend.

Reply:
Let me change it to just line breaks then!

Reply:
Actually the tricky thing here is that this feedback column is also for user-inputted reviews, and it seems like we wanted to support bullet points?

Reply:
closing this for quicker progress for now. Can revisit.

Reply:
Understood, thanks

Reply:
I am asking product/design for clarifications on this.